ui/lib/core/addon/templates/components/replication-action-disable.hbs

```diff
@@ -1,67 +1,45 @@
-<h4 class="title is-5 is-marginless">
-  Disable Replication
-</h4>
-<div class="content">
-  <p>
-    Disable {{replicationDisplayMode}} Replication entirely on the cluster.
-    {{#if model.replicationAttrs.isPrimary}}
```

Comment:
Matching the designs for this component means that much of this additional context was lost. We may want to add some of this back into the confirmationModal